src/core/strategy_coordinator.py

```
@@ -908,9 +936,25 @@ async def process_strategy_signal(self, strategy_name: str, signal: Dict) -> Dic
+            features = enriched_signal.get('features', {})
+            quality_features = {
+                "ml_component": enriched_signal.get("ml_confidence"),
+                "volume_component": features.get("volume_score") or enriched_signal.get("volume_24h"),
+                "momentum_component": features.get("momentum") or enriched_signal.get("momentum"),
+                "spread_component": features.get("spread") or enriched_signal.get("spread")
+            }
+            quality_result = compute_quality(quality_features, logger)
+            enriched_signal["quality_score"] = quality_result["value"]
+            enriched_signal["quality_breakdown"] = quality_result
+
             # Adım 7: Sinyali ve Rota Bilgisini Hazırla
```

Ruff (0.14.6)

Comment contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF003)